ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -1750,6 +1750,7 @@ void HqlCppTranslator::cacheOptions()
         DebugOption(options.implicitGroupHashDedup,"implicitGroupHashDedup",false),
         DebugOption(options.reportFieldUsage,"reportFieldUsage",false),
         DebugOption(options.reportFileUsage,"reportFileUsage",false),
+        DebugOption(options.recordFieldUsage,"recordFieldUsage",false),
```

Comment:
We need to remember to document this new #option as an ECL requirement for workunits that will be controlled by column level security